README.md

```diff
@@ -8,14 +8,14 @@
 This repository is a part of the Shotgun Pipeline Toolkit.
 
 - For more information about this app and for release notes, *see the wiki section*.
-- For general information and documentation, click here: https://support.shotgunsoftware.com/entries/95441257
+- For general information and documentation, click here: https://developer.shotgridsoftware.com/e95205a8/?title=An+Overview+of+Toolkit
```

Comment:
Shouldn't be better to use the User Guide URL?
https://developer.shotgridsoftware.com/d587be80/?title=Integrations+User+Guide